README.md

```diff
@@ -5,7 +5,7 @@ This repo is the Java client library for integrating with Posten signering. Plea
 
 ## 📕 Documentation
 
-Integration guide can be found at [digipost.github.io/signature-api-client-java](http://digipost.github.io/signature-api-client-java).
+Get started by [reading the documentation](http://signering-docs.rtfd.io/).
```

Comment:
RTFD? Really? 😆

Reply:
Hahaha. Ja. `ReadTheDocs` og `rtfd` er synonymer i denne sammenhengen. Men vet ikke hva forkortelsen er. Jeg tipper det er Read The Fucking Docks ;p